proposals/src/tests/test_main.py

```diff
@@ -191,55 +191,55 @@ class TestCreateOrUpdateProposal:
         },
     ]
 
+    expeted_measurement_periods = [
```

Comment:
```suggestion
    expected_measurement_periods = [
```
typo (also below)

Reply:
I fixed it in a later PR, I'll keep the typo here as it simplifies rebasing later